enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -0,0 +1,249 @@
+- configure the SD internal audit policy
+- configure API audit policy and forwarding using name-spaced logging customer resources in the management cluster.
+  - for example: Red Hat Service Delivery team forwards audit logs to internal splunk destination.
+- independent of user configuration.
+
+### Goals
+
+- Multiple audit policies can be defined in a management cluster, independent of hosted cluster policies.
+- Multiple audit policies can be defined in a hosted cluster, independent of the management cluster policies.
+  See [api-audit-log-policy](/api-audit-log-policy.md) for how the CLO will support this.
+- Allow some forwarding in the _management_ cluster to be configured from the _hosted_ cluster.
+- Keep direct control of management cluster forwarding on the management cluster.
+  - hosted cluster has no direct access to control plane configuration.
+  - data plane resources can request forwarding by the management cluster.
+  - management cluster HLO automatically validates data-plane requests, and enables them if they are safe.
+
+### Non-Goals
+
+- This proposal only covers API audit logs. No change to application, infrastructure or node audit logs.
+- Management cluster is not _required_ to honour all possible forwarding configurations allowed by CLF. \
+  Whether or how to restricting management cluster forwarding is an operational decision.
+
+## Proposal
+
+### Workflow Description
+
+#### Hosted Cluster, data plane
+
+1. Deploy the CLO as usual.
+1. Create a `Policy.audit.k8s.io` resource with the desired policy
+   (see [api-audit-log-policy](/api-audit-log-policy.md))
+1. Create a `HypershiftLogForwarder` (HLF) resource with an `audit` input linked to the policy. \
+   The `HypershiftLogForwarder` API is a restricted subset of the `ClusterLogForwarder` API
+1. Create HLF `outputs` and `pipelines` to forward the resulting logs.
+1. Optionally create a `ClusterLogForwarder` to forward non-API logs as usual.
+
+#### Management Cluster
+
+1. Deploy the CLO, all name-spaces.
+1. Deploy the new Hypershift Logging Operator (HLO), watching all namespaces.
+1. Create a single `HypershiftLogWatcher` resource containing a template for HCP logging.
+
+See "Implementation Details" for more details.
+
+### API Extensions
+
+#### HypershiftLogForwarder
+
+The `HyperShiftLogForwarder` resource is created on the data plane (hosted cluster),
+but is reconciled by the HLO on the _management_ cluster.
+
+This allows the service consumer to provide forwarding configuration,
+but leaves the service provider in control of whether to implement it in the HCP.
+
+`HyperShiftLogForwarder` is a subset of the `ClusterLogForwarder` API:
+- Only the `audit` input is allowed, and only API server audit logs will be forwarded.
+- Any pipeline/output features that cannot be used with an `audit` input are disabled.
+- The service provider MAY impose additional restrictions on supported log targets, security restrictions etc.
```

Comment:
1. what are the base set of output types that is supported in HLF?
2. how does the service provider restrict the supported output types?

SRE needs splunk as an output.  If we know this is in scope I would like it listed so that is clear.  Also from customers I am hearing CloudWatch and syslog.

Reply:
I will remove the talk of out restrictions, if it turns out we do need such restrictions we can introduce them later. 
It was an idea I had at the beginning but it doesn't seem to be really necessary.